app/src/play/java/com/duckduckgo/referral/PlayStoreAppReferrerStateListener.kt

```diff
@@ -173,8 +174,12 @@ class PlayStoreAppReferrerStateListener @Inject constructor(
                 variantManager.updateAppReferrerVariant(result.campaignSuffix)
                 appReferrerDataStore.campaignSuffix = result.campaignSuffix
             }
-            is EuAuctionReferrerFound -> {
-                variantManager.updateAppReferrerVariant(RESERVED_EU_AUCTION_VARIANT)
+            is EuAuctionSearchChoiceReferrerFound -> {
+                variantManager.updateAppReferrerVariant(RESERVED_EU_SEARCH_CHOICE_AUCTION_VARIANT)
+                appReferrerDataStore.installedFromEuAuction = true
```

Comment:
do we need to differentiate between the cases here?

`appReferrerDataStore.installedFromEuAuction = true`

☝️ this is set whether from browser choice or search choice routes, and i'm curious if we need to differentiate between them? Might not matter because the important part (updating variant) happens, but wanted to flag in case.

Reply:
@CDRussell good shout! This is used for the affiliate in SERP. As of now there’s no need to differentiate them, but if needed we can get it changed.